lib/features/email/presentation/email_view.dart

```diff
@@ -1,16 +1,21 @@
 import 'package:core/core.dart';
+import 'package:flutter/cupertino.dart';
 import 'package:flutter/material.dart';
 import 'package:get/get.dart';
 import 'package:model/model.dart';
+import 'package:tmail_ui_user/features/email/domain/state/get_email_content_state.dart';
 import 'package:tmail_ui_user/features/email/presentation/email_controller.dart';
 import 'package:tmail_ui_user/features/email/presentation/widgets/app_bar_mail_widget_builder.dart';
+import 'package:tmail_ui_user/features/email/presentation/widgets/attachment_file_tile_builder.dart';
 import 'package:tmail_ui_user/features/email/presentation/widgets/bottom_bar_mail_widget_builder.dart';
-import 'package:tmail_ui_user/features/mailbox_dashboard/presentation/mailbox_dashboard_controller.dart';
+import 'package:tmail_ui_user/features/email/presentation/extensions/email_content_extension.dart';
+import 'package:tmail_ui_user/features/email/presentation/widgets/message_content_tile_builder.dart';
+import 'package:tmail_ui_user/features/email/presentation/widgets/sender_and_receiver_information_tile_builder.dart';
 import 'package:tmail_ui_user/main/localizations/app_localizations.dart';
 
-class EmailView extends GetWidget<EmailController> {
+class EmailView extends GetWidget {
```

Comment:
Why not binding `EmailController` to `EmailView`?

Reply:
On mobile devices EmailView will be canceled when exiting the screen. If you attach an EmailController to it, the controller will be destroyed. On the tablet screen it won't find it.

Reply:
> On mobile devices EmailView will be canceled when exiting the screen. If you attach an EmailController to it, the controller will be destroyed

yes, but I think it is true, why we need to care to the controller whenever the view is disposed?

Reply:
> On the tablet screen it won't find it.

In the tablet, the view still here, why it is not found?

Reply:
Because it is destroyed when we call Get.back(); on mobile

Reply:
on mobile, when open another mail, why the controller is not re-create?

Reply:
If we do so, we will have to re-create in some cases such as switching between tablet and mobile.